tests/settings/SettingTest.py

```diff
@@ -8,7 +8,7 @@
     Setting, path, path_list, url, typed_dict, typed_list, typed_ordered_dict,
     glob, glob_list,
     language,
-)
+    float_list, bool_list, int_list, str_list)
```

Comment:
maybe you should keep the old code-style for imports, so leaving the closing `)` on a new line and having a trailing comma:
```
    language,
    float_list, bool_list, int_list, str_list,
)
```